.gitignore

```
@@ -141,4 +141,7 @@ dmypy.json
 cython_debug/
 
 #PyCharm nonsense
-.idea/
\ No newline at end of file
+.idea/
+
+#Error log format
+*.log
```

```suggestion
*.log

```
Could you add a newline to the end of the file? This helps with repo bloat over time